community/common/src/main/java/org/neo4j/kernel/api/exceptions/Status.java

```diff
@@ -408,6 +409,24 @@ public Code code()
         }
     }
 
+    enum Session implements Status
+    {
+        SessionTerminated( ClientError, "The session has been terminated by the server." );
```

Comment:
Suggest something like `InvalidSession( ClientError, "The session is no longer available, possibly due to termination." )`. This describes the effect/outcome rather than the action that led to this outcome and as such is a more future-proof description.